test/e2e/admission.go

```diff
@@ -33,13 +33,12 @@ var _ = Describe("Job E2E Test: Test Admission service", func() {
 
 	It("Default queue would be added", func() {
 		jobName := "job-default-queue"
-		namespace := "test"
-		context := initTestContext(options{})
-		defer cleanupTestContext(context)
+		ctx := initTestContext(options{})
```

Comment:
let's keep this var name.

Reply:
`context` is a golang standard lib, so let's change this confusing name